app/views/check_in/check_ins.html.erb

```diff
@@ -0,0 +1,18 @@
+<section id="check-in">
+  <h3>Please Check In</h3>
+  <p><%= current_account.project%></p>
+
+  <%= f.label :what_day_is_this_for %>
+  <%= f.text_field :worked_at, class: "datepicker" %>
+  <%= render partial: "error_for_field", locals: { model: @log, field: :worked_at } %>
+
+  <%= f.label :quality, "How'd it go?" %>
+  <%= f.select :quality, options_for_select(CheckInHelper::ATTENTION_QUALITY, @log.quality) %>
+  <%= render partial: "error_for_field", locals: { model: @log, field: :quality } %>
+
+  <%= f.label :notes %>
+  <%= f.text_area :notes %>
+  <%= render partial: "error_for_field", locals: { model: @log, field: :notes } %>
+
```

Comment:
Also, we'll need to use nested_attributes for this; since a CheckIn object will `have_many :logs`: http://api.rubyonrails.org/classes/ActiveRecord/NestedAttributes/ClassMethods.html

You will likely need to define the CheckIn model a bit for this to work.

In the near term, I would use `form_tag` and `label_tag` instead of `f.` until we have the model stuff working; as `form_for` and `fields_for` expect a working ActiveRecord model; which we don't7 have yet.